cinema/views.py

```diff
@@ -31,9 +35,29 @@ class CinemaHallViewSet(viewsets.ModelViewSet):
 
 
 class MovieViewSet(viewsets.ModelViewSet):
-    queryset = Movie.objects.all()
+    queryset = Movie.objects.prefetch_related("genres", "actors")
     serializer_class = MovieSerializer
 
+    @staticmethod
+    def params_to_int(query_string):
+        return [int(num) for num in query_string.split(",")]
```

Comment:
The `params_to_int` method assumes valid input, which might not handle invalid input gracefully. Consider adding error handling for cases where the input might not be valid integers to improve the robustness of your code.